addons/mail/models/mail_tracking_value.py

```diff
@@ -6,6 +6,28 @@
 from odoo import api, fields, models, tools
 
 
+class IrModelField(models.Model):
```

Comment:
Small note: there is already an ir model fields override in ir_model.py .

How to gain 22 points: split file into 2 (becayse @tivisse is too shy to do it)

How to gain 222 points: make a small test